.github/workflows/example-scan.yaml

```diff
@@ -9,7 +9,7 @@ jobs:
       security-events: write
     steps:
     - uses: actions/checkout@v3
-    - uses: kubescape/github-action@main
+    - uses: kubescape/github-action@feat-image-scanning
```

Comment:
chore: be aware that we cannot merge the action with these changes

These have to point to the main branch at least for the time being. Ideally, they should be pinned to a version.